### DISCUSSION TITLE
Open in New Window or Tab

### DISCUSSION THREAD

**Discussion by yodaphone** [alexjustesen/speedtest-tracker#490] [Ideas]
in the General Settings page, when you want to change the CRON schedule and click on "HERE" the Cron expression generator opens in the same page. It also happens when you click on "Time Format" options. please change it so that it opens in a new TAB or Window

**Comment by alexjustesen** [comment]
This field takes markdown in the framework which doesn't support new tab or window at this time.

https://filamentphp.com/docs/2.x/forms/fields#helper-messages-and-hints
